# Defines the 3-D bounding box of a vehicle.
# The message definition is compatible with the Carla Cpp API carla::geom::BoundingBox.
geometry_msgs/Point extent
geometry_msgs/Point location
